pennylane/plugins/tests/conftest.py

```diff
@@ -79,6 +79,40 @@ def _skip_if(dev, capabilities):
     return _skip_if
 
 
+@pytest.fixture(scope="session")
+def skip_if_ops():

```

Comment:
Minor point: is this a clear name?
Would something like 
`skip_if_ops_unsupported`? It's longer, but clearer (to me at least!)